AI-and-Analytics/End-to-end-Workloads/LidarObjectDetection-PointPillars/README.md

```diff
@@ -49,17 +49,32 @@ To build and run the PointPillars sample, the following libraries have to be ins
 3. Boost (including `boost::program_options` and `boost::filesystem` library). For Ubuntu, you may install the libboost-all-dev package.
 4. Optional: If the sample should be run on an Intel GPU, it might be necessary to upgrade the corresponding drivers. Therefore, please consult the following page: https://github.com/intel/compute-runtime/releases/
 
+### Using Visual Studio Code*  (VS Code)
+
+You can use VS Code extensions to set your environment, create launch configurations,
+and browse and download samples.
+
+The basic steps to build and run a sample using VS Code include:
+ - Download a sample using the extension **Code Sample Browser for Intel oneAPI Toolkits**.
+ - Configure the oneAPI environment with the extension **Environment Configurator for Intel oneAPI Toolkits**.
+ - Open a Terminal in VS Code (**Terminal>New Terminal**).
+ - Run the sample in the VS Code terminal using the instructions below.
+
```

Comment:
@bdmoore1 -- I think it would be useful to provide a link to the VS Code marketplace that points to all the Intel offerings, that way if an extension name changes, a new extension is added, etc. the current information will always be presented. For example:

> The complete set of Intel VS Code extensions can be found [in the VS Code marketplace](https://marketplace.visualstudio.com/publishers/intel-corporation).

Also, I recommend changing the last bullet to:

> Build and run the sample in the VS Code terminal using the instructions below.